src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/Setup/CertificateUtility.cs

```diff
@@ -308,5 +305,28 @@ public static void ChangeServerTceSetting(bool fEnable, SqlConnectionStringBuild
                 }
             }
         }
+
+        private static void ClearCache(MemoryCache cache)
+        {
+            // Get the Clear method of the cache and use it if available. This is available in .NET 8.
```

Comment:
Seems like will be available in other versions: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.caching.memory.memorycache.clear?view=net-8.0#applies-to